src/Portalum.Zvt/ZvtClientConfig.cs

```diff
@@ -26,5 +26,13 @@ public class ZvtClientConfig
         /// Timeout after Command Acknowledge and Completion
         /// </summary>
         public TimeSpan CommandCompletionTimeout = TimeSpan.FromSeconds(180);
+
+        
+        /// <summary>
+        /// Maximum number of times the AskForCompletionInfo callback is called. After this number of
+        /// times a auto reversal will be triggered and the payment will fail. A PT usually queries
+        /// every 2 (ZVT standard) to 4 (observed in practice) seconds for the completion status.
+        /// </summary>
+        public byte GetAsyncCompletionInfoLimit = 100;
```

Comment:
altough the ZVT default states infinite is the default value, i have observed that a CardComplete Valina terminal only asks 3 times.